getting started/requests.md

```diff
@@ -0,0 +1,80 @@
+# Requests URL
+
```

Comment:
The page is made only for authorization. Other requests should be made in separate files.

Reply:
Actually, the file is called `requests.md` :)

Reply:
It's a reason for changing a name. It's not a good way, to manipulate with one file simultaneously for different APIs.
Let's create several separate files for it instead